src/PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+spectre_target_sources(
+  GeneralRelativity
+  PRIVATE
+  DerivSpatialMetric.cpp
+  ExtrinsicCurvature.cpp
+  GaugeSource.cpp
+  Phi.cpp
+  Pi.cpp
+  SpacetimeDerivativeOfSpacetimeMetric.cpp
+  SpacetimeDerivOfDetSpatialMetric.cpp
```

Comment:
alphabet - `DerivOf` should get sourted to before `Derivative` because caps < lower case